tests/subsys/cpp/cxx/src/main.cpp

```diff
@@ -17,16 +17,62 @@
 #include <device.h>
 #include <kernel.h>
 #include <net/buf.h>
-#include <sys/byteorder.h>
+/* #include <sys/byteorder.h> conflicts with __bswapXX on native_posix */
```

Comment:
Do we have a separate issue for this, for tracking?

Reply:
@cfriedt See https://github.com/zephyrproject-rtos/zephyr/issues/44324#issuecomment-1096271561